build/config/ios/ios_sdk.gni

```diff
@@ -21,6 +21,9 @@ declare_args() {
 
   # The path to the iOS simulator SDK.
   ios_simulator_sdk_path = ""
+
+  # Version of iOS that we're targeting for tests.
+  ios_testing_deployment_target = "13.0"
```

Comment:
The variables here apply to all targets (ldflags from the variables defined here are specified in `build/toolchain/mac/BUILD.gn`). Is there a reason tests can't be run on older simulators? In that case, you can just modify the toolchain definition.

Reply:
That would require us to edit OCMock's source code.  The variable is meant to be a mirror to `ios_deployment_target` so it seems like it should be used the same way.

Reply:
I see, so its enforced by a dependency. `ios_deployment_target` is used in `//build/toolchain/mac/BUILD.gn` which applies that variable to all iOS targets. This is unlike  `ios_testing_deployment_target` here which targets need to apply themselves (like in the case of `ocmock`).

Eh, this is only used in target. We can figure out how to make this consistent later.